src/Core/Domain/SpecificPrice/CommandHandler/AddSpecificPriceHandlerInterface.php

```diff
@@ -26,11 +26,13 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\SpecificPrice\CommandHandler;
 
+use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\CommandHandler\AddProductSpecificPriceHandlerInterface;
 use PrestaShop\PrestaShop\Core\Domain\SpecificPrice\Command\AddSpecificPriceCommand;
 use PrestaShop\PrestaShop\Core\Domain\SpecificPrice\ValueObject\SpecificPriceId;
 
 /**
- * Interface for handling AddSpecificPrice command
+ * @deprecated since 1.7.8.0
```

Comment:
```suggestion
 * @deprecated since 1.7.9.0
```

Reply:
these (related to Add{command}) were already deprecated in 178, just annotations were missing. (see previous jolelievre comments)